src/viur/core/module.py

```diff
@@ -43,8 +43,9 @@ def __init__(self, func: t.Callable):
         self.signature = inspect.signature(self._func)
 
         # Guards
-        self.skey = None
+        self.skey = False
         self.access = None
+        self.decorators = []
```

Comment:
I would like to call these "guards", instead of "decorators".
```suggestion
        self.guards = []
```